packages/ember-glimmer/tests/integration/outlet-test.js

```diff
@@ -1,46 +1,50 @@
 import { RenderingTest, moduleFor } from '../utils/test-case';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
+import { runAppend } from 'ember-runtime/tests/utils';
 
 moduleFor('outlet view', class extends RenderingTest {
   constructor() {
     super(...arguments);
 
     let CoreOutlet = this.owner._lookupFactory('view:-outlet');
-    this.top = CoreOutlet.create();
+    this.component = CoreOutlet.create();
   }
 
-  teardown() {
-    runDestroy(this.top);
-
-    super.teardown(...arguments);
-  }
-
-  withTemplate(string) {
-    return {
+  ['@htmlbars should render the outlet when set after DOM insertion']() {
```

Comment:
This test title is wrong now, since when it appends to the DOM it has a fully populated  outletState.